cookbooks/tile/recipes/default.rb

```diff
@@ -161,12 +161,9 @@
   python3-mapnik
   python3-pyproj
   python3-setuptools
+  python3-pyotp
```

Comment:
Can we move this up so it's in alphabetical order like the others?

Reply:
done